packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -837,12 +837,6 @@ export function createLodestarMetrics(
       }),
 
       // Validator Monitor Metrics (per-epoch summaries)
-      // Only track prevEpochOnChainBalance per index
-      prevEpochOnChainBalance: register.gauge<"index">({
```

Comment:
I'm really not sure about removing this if it does the same as the newly added metric